src/app/[locale]/_component/header/header.tsx

```diff
@@ -1,10 +1,42 @@
 'use client';
 import { HeaderAccount, HeaderProduct, LogoIOApp } from '@pagopa/mui-italia';
 import React from 'react';
+import ExitToAppIcon from '@mui/icons-material/ExitToApp';
+import ManageAccountsIcon from '@mui/icons-material/ManageAccounts';
 import useLogin from '../../_hooks/useLogin';
+import useLocalePush from '../../_hooks/useLocalePush';
+import { ROUTES } from '../../_utils/routes';
 
 const Header = (): React.ReactElement => {
-  const { userLogged, isLoggedIn } = useLogin();
+  const { userLogged, isLoggedIn, logOut } = useLogin();
+  const pushWithLocale = useLocalePush();
+  const JWT_SPID_LEVEL_L1 = process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1;
+
+  const userMenuActionsBasic = [
+    {
+      id: 'logout',
+      label: 'Esci dal portale',
+      onClick: () => {
+        logOut();
+      },
+      icon: <ExitToAppIcon fontSize="small" color="inherit" />,
+    },
+  ];
+
+  const userMenuActions =
+    isLoggedIn && userLogged?.spidLevel !== JWT_SPID_LEVEL_L1
+      ? [
+          {
+            id: 'profile',
+            label: 'Vai al profilo',
+            onClick: () => {
+              pushWithLocale(ROUTES.PROFILE);
+            },
+            icon: <ManageAccountsIcon fontSize="small" color="inherit" />,
+          },
+          ...userMenuActionsBasic,
+        ]
+      : userMenuActionsBasic;
```

Comment:
```suggestion
  const userMenuActions =
    isLoggedIn && userLogged?.spidLevel !== JWT_SPID_LEVEL_L1
      ? [
           ...userMenuActionsBasic,
          {
            id: 'profile',
            label: 'Vai al profilo',
            onClick: () => {
              pushWithLocale(ROUTES.PROFILE);
            },
            icon: <ManageAccountsIcon fontSize="small" color="inherit" />,
          },
        ]
      : userMenuActionsBasic;
```

Reply:
When using the rest (`...`) operator, it's a good practice to put it at the top, so that to be sure the additional propertyies won't be overridden by a property inside the object used with the rest operator. What do you think?

Reply:
Ah, okay. I got the point.  So I leave the choice to you.  If we left the way it is, it's important to understand the reason why we're using the rest operator at the bottom. If we wanto to add a sorting feature we could think about a memoizing approach to avoid sorting at every re-rendering of the page.

Reply:
Ok, fixxed using useMemo.